lib/s3.go

```diff
@@ -60,7 +60,14 @@ func newRemoteStore(cfg Config, logger printer) (*s3Store, error) {
 		}
 	}
 
-	s = &s3Store{svc: s3.New(sess), cfc: cfc, publicReadACL: cfg.PublicReadACL, bucket: cfg.BucketName, r: cfg.conf.Routes, bucketPath: cfg.BucketPath}
+	acl := "private"
+	if cfg.ACL != "" {
```

Comment:
With the next major version, we could simplify this to just `acl := cfg.ACL` and default to `private` in the config.